package/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.11.3
+FROM alpine:3.11.11
```

Comment:
Any reason not to push to 3.13 ? Just stability ? We'll probably migrate this later anyway so its fine.

Reply:
Yes, stability. The issues were fixed in this patch, so I went with it.